packages/flutter/lib/src/material/app_bar.dart

```diff
@@ -1115,14 +1115,19 @@ class _AppBarState extends State<AppBar> {
       appBar = Column(
         mainAxisAlignment: MainAxisAlignment.spaceBetween,
         children: <Widget>[
-          Flexible(
-            child: ConstrainedBox(
-              constraints: BoxConstraints(maxHeight: toolbarHeight),
-              child: appBar,
-            ),
+          if (toolbarHeight == 0)
```

Comment:
This seems really specific to the use case in the issue. What if the toolbar height is 20? or 100? Or 1?

Reply:
Can you provide context on this?